app/page.tsx

```diff
@@ -0,0 +1,122 @@
+          <article className="mx-6 mb-20 mt-10 h-full w-full max-w-[52rem] md:mx-10 lg:mx-16">
+            <HomeTitle title="Welcome" description="" />
+            <p>
+              Welcome to the IceRPC documentation! You will find on this site a
+              wide range of materials, including installation instructions,
+              tutorials, programming guides, and more.
+            </p>
+            <p>
+              Want to make RPCs without reinventing the wheel? Want to take
+              advantage of QUIC through a convenient, modern API? You&apos;ve
+              come to the right place.
+            </p>
+
+            <p>
+              IceRPC is currently available for C# / .NET. We plan on adding
+              support for more languages soon, starting with Rust.
+            </p>
+            <Divider />
+            <HomeTitle title="Key Features" description="Discover IceRPC" />
+            <Table>
+              <thead>
+                <TR>
+                  <TH>Feature</TH>
+                  <TH>Description</TH>
+                </TR>
+              </thead>
+              <tbody>
+                <TR>
+                  <TD>
+                    <AppLink href="/icerpc/invocation/invocation-pipeline">
+                      Invocation pipeline
+                    </AppLink>
+                  </TD>
+                  <TD>
+                    Customize your client-side processing by composing
+                    interceptors into an invocation pipeline.
+                  </TD>
+                </TR>
+                <TR>
+                  <TD>
+                    <AppLink href="/icerpc/dispatch/dispatch-pipeline">
+                      Dispatch pipeline
+                    </AppLink>
+                  </TD>
+                  <TD>
+                    Compose middleware and services into your own custom
+                    server-side dispatch pipeline.
+                  </TD>
+                </TR>
+                <TR>
+                  <TD>
+                    <AppLink href="/icerpc/dispatch/router">
+                      Path-based routing
+                    </AppLink>
+                  </TD>
+                  <TD>
+                    Learn how to route a request to the right service based on
+                    the request&apos;s path.
+                  </TD>
+                </TR>
+                <TR>
+                  <TD>
+                    <AppLink href="/slice">Slice</AppLink>
+                  </TD>
+                  <TD>
+                    A modern IDL and serialization format developed in tandem
+                    with IceRPC.
+                  </TD>
+                </TR>
+                <TR>
+                  <TD>
+                    <AppLink href="/icerpc/dependency-injection/di-and-icerpc-for-csharp">
+                      Slice
+                    </AppLink>
+                  </TD>
+                  <TD>Dependency injection</TD>
```

Comment:
This entry in the table got mangled somehow.
```suggestion
                  <TD>
                    <AppLink href="/icerpc/dependency-injection/di-and-icerpc-for-csharp">
                      Dependency injection
                    </AppLink>
                  </TD>
                  <TD>Learn how to use IceRPC with a DI container.</TD>
```